src/docs/Foundations/Layout/Container/code.mdx

```diff
@@ -1,6 +1,7 @@
 ---
-title: 'Container'
+title: 'Code'
 order: 2
+description: 'The container define the main content width as well as the minimum negative space between the horizontal edges of the screen and the content.''
```

Comment:
Additional single quote
`description: 'The container define the main content width as well as the minimum negative space between the horizontal edges of the screen and the content.'`